### DISCUSSION TITLE
[Bug]: What does stable zap gun do exactly? and how do I update the mod without having to reinstall

### DISCUSSION THREAD

**Discussion by joep26020** [winstxnhdw/lc-hax#80] [Q&A] [answered]
### What happened?

asdsasdad

### Current Commit Hash

asdasda

### Injector

- [ ] SharpMonoInjectorCore
- [ ] Others

### Acknowledgement

- [X] I have **recursively** cloned the repository as instructed in the [README](https://github.com/winstxnhdw/lc-hax/edit/main/README.md)
- [X] I have done my due diligence to check for similar issues

**Comment by 1-Mythical** [comment, marked as answer]
When you use Zap Gun, the beam that stuns the enemies will drift off left or right and you would need to move your mouse the opposite way to keep it stablized. This basically just removed the drift.

Just run ```git pull``` in terminal to update. Any local changes you made must either be reverted or you can run this command: ```git reset --hard HEAD```

**Comment by winstxnhdw** [comment]
You're the best @MythicalTrashcan
